### PR TITLE
Add user configs to gitignore

### PR DESCRIPTION
**Pull request by xwu99**
Ignore configs

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -14,5 +14,9 @@ report/
 .classpath
 .project
 .settings/
+conf/benchmarks.lst
+conf/frameworks.lst
+conf/hibench.conf
```
Comment (carsonwang) [inline .gitignore:19]: These files are already in git. Will that be a problem if someone wants to modify these files?

Comment (xwu99) [reply .gitignore:19]: When switching from one branch to another, you can still use the same config. Don't bother to resolve conflicts when switching branch. Just ignore the changes to config. 
Also can avoid accidently commit user-modified configs to repo when you modify those files to benchmark something.   You can still use git add -f to add if really need to commit the change.

Comment (xwu99) [reply .gitignore:19]: Just like hadoop.conf and spark.conf is user specific config, no need to commit. does it make sense?

Comment (carsonwang) [reply .gitignore:19]: Ok.

**Pull request merged**